ds/ds.php

```diff
@@ -35,7 +35,7 @@ public function clear(): void;
         /**
          * Returns a shallow copy of the collection.
          * @link https://www.php.net/manual/en/ds-collection.copy.php
-         * @return Collection
+         * @return \Ds\Collection
```

Comment:
What is the purpose of such changes? type is already being properly inferred as `\Ds\Collection` 
<img width="410" alt="Screenshot 2020-12-16 at 01 32 56" src="https://user-images.githubusercontent.com/37301285/102280861-f2dca200-3f3e-11eb-8a2a-dcdfd3fd0f1a.png">

Reply:
Yes via PhpStorm, but it's easier for everyone if there are unambiguous phpDocs, and so a simple script can check the phpDocs, without the need to resolve the namespaces.

Reply:
@voku I agree with @isfedorov that the change is unneeded and should be reverted. It makes PHPDoc more verbose and any script will have to resolve namespaces to be correct anyway since it could be easily be broken by some other change. 
Let's not simplify the creation of some abstract scripts by complicating existing stubs and forcing addition rules (only FQN in PHPDoc).